helm/blazegraph/templates/RWStore.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "blazegraph.fullname" . }}-rwstore
+  labels:
+    app: {{ template "blazegraph.name" . }}
+    chart: {{ template "blazegraph.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  rwstore: |-
```

Comment:
due to the non-changing nature of this would it make sense to have this in the container itself and not as a mounted config map?

Reply:
I was not sure about this one. It was mostly a drop in from the sysalt repo. I don't disagree that it may make more sense to live within the repo. I can move it now or later.

Reply:
I'll leave it up to you, but I would move this entire PR to the docker-blazegraph repo and put the configmap into the container.